PSKoans/Koans/Advanced Techniques/AboutRedirection.Koans.ps1

```diff
@@ -3,7 +3,140 @@
 param()
 <#
     Redirection
+
+    The PowerShell streams are numbered in order from 1-6.
+
+    Stream #	Description	        Introduced in
+    --------	-----------	        -------------            
+    1	        Success Stream	    PowerShell 2.0
+    2	        Error Stream	    PowerShell 2.0
+    3	        Warning Stream	    PowerShell 3.0
+    4	        Verbose Stream	    PowerShell 3.0
+    5	        Debug Stream	    PowerShell 3.0
+    6	        Information Stream	PowerShell 5.0
+    *	        All Streams	        PowerShell 3.0
+
+    https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_redirection?view=powershell-6
 #>
 Describe 'Redirection Operators' {
+    BeforeAll {
+        $OriginalDebugPreference = $DebugPreference
+        $OriginalVerbosePreference = $VerbosePreference
+        $FilePath = 'TestDrive:\TestFile.txt'
+
+        $DebugPreference = $VerbosePreference = 'Continue'
+    }
+
+    It 'can be used to redirect output to files' {
+        # Providing a path will send output to that location, creating the file if necessary.
+        'I see mountains once again as mountains,' > $FilePath
+        $FileContent = Get-Content -Path $FilePath
+
+        '__' | Should -Be $FileContent
+    }
+
+    It 'can append to files' {
+        # Appending to existing files adds a new line.
+        'And waters once again as waters.' >> $FilePath
+        $FileContent = Get-Content -Path $FilePath
+
+        @( 'I see mountains once again as mountains,', '__' ) | Should -Be $FileContent
+    }
+    
+    It 'can use numbered streams' {
+        Write-Output 'The stream flows forth with successes!' 1> $FilePath
+        $FileContent = Get-Content -Path $FilePath
+        '__' | Should -Be $FileContent
+        
+        & {Write-Error 'The gasoline is in the water.'} 2> $FilePath
```

Comment:
A brief comment indicating the reason this is phrased differently would be a good idea, to minimise confusion.